src/pal/inc/pal.h

```diff
@@ -5222,6 +5222,22 @@ DWORD
 PALAPI
 EnsureOpenSslInitialized();
 
+PALIMPORT
+DWORD
+PALAPI
+ForkAndExecProcess(
```

Comment:
We should create a new header files for the corefx helpers - something like pal_corefx.h